layouts/index.html

```diff
@@ -224,9 +221,9 @@ <h5>Researchers</h5>
             </div>
             <h5>Developing world</h5>
             <p>
-              High-latency networks are a major barrier of entry to the developing
-              world. IPFS provides resilient access to data, independent of low
-              latency or connectivity to the backbone.
+              High-latency networks are a big barrier for the developing
+              world. IPFS provides resilient access to data independent of
+              latency or backbone connectivity.
```

Comment:
Don't need to fix in this PR, but at some point we should change `developing world` to something less culturally judgemental, favoring a description of the specific problem, eg `for people with poor internet infrastructure`.

Reply:
Changed first line to "High-latency networks are a big barrier for those with poor internet infrastructure." Could change headline to "Offline-friendly" rather than "Developing world", but that doesn't seem quite right either. Will continue to mull it over 😊